### DISCUSSION TITLE
退出缩小到托盘，优化切换倍速

### DISCUSSION THREAD

**Discussion by lVentus** [ywmoyue/biliuwp-lite#757] [feature]
### 问题描述

是否考虑实现关闭时不退出，而是缩小到托盘的功能？这样下次打开时不用等待程序启动。
此外，长按倍速播放在开始和结束时，视频是流畅切换的，但音频会卡一下。
谢谢大大维护！

### 描述解决方案

关闭时不退出，而是缩小到托盘
优化倍速播放

### 代码贡献

否

**Comment by ywmoyue** [comment]
缩小到托盘这个不做，现在播放视频有内存泄漏问题，长期不重启应用会占内存

**Comment by ywmoyue** [comment]
切换播放倍速这个现在也没什么好的优化方法

**Reply by lVentus** [reply]
好吧，谢谢大大答复

**Comment by ManyullynSword** [comment]
我也挺想要这个功能的······
